kedro/config/omegaconf_config.py

```diff
@@ -147,15 +161,21 @@ def __getitem__(self, key) -> Dict[str, Any]:
         read_environment_variables = key == "credentials"
 
         # Load base env config
-        base_path = str(Path(self.conf_source) / self.base_env)
+        if self._protocol == "file":
+            base_path = str(Path(self.conf_source) / self.base_env)
+        else:
+            base_path = str(Path(self._fs.ls("")[0]) / self.base_env)
```

Comment:
Might be worth also documenting the expected structure of the compressed config file, for eg, when I archive the conf folder with `shutil.make_archive("zip_conf", "zip", "conf)`, the file structure is -
```
zip_conf.zip
|_ README.md
|_ base
|_ local
```
The error message I got is `MissingConfigException: Given configuration path either does not exist or is not a valid directory: README.md/base`.

Reply:
Done!